extras/multisigs.json

```diff
@@ -84,6 +84,12 @@
     "lm": "0x040E995520F92F96142d1a76c16D4af21A2eFDE7",
     "maxi_ops": "0x040E995520F92F96142d1a76c16D4af21A2eFDE7"
   },
+  "fantom": {
+    "dao": "0xFe2022da08d17ac8388F5bEFE4F71456255105A4"
+  },
+  "bnb": {
+    "dao": "0x58099b94e660bBe19848547F6c5d76DcA7282E45"
```

Comment:
Where did you obtain this information? Couldn't verify that address from the docs or other resources: https://docs.balancer.fi/concepts/governance/multisig.html#the-multisigs-and-their-addresses

Reply:
via https://snapshot.org/#/balancer.eth/proposal/0xbd2f095f693ec886a4774f928656a6552632f32ec81971b1072b569af59d91d5